src/main/java/lotto/domain/LottoWinningCondition.java

```diff
@@ -18,11 +18,11 @@ void checkLottoWinningCondition(Lotto previousLotto, LottoNumber bonusNumber) {
         }
     }
 
-    Lotto getPreviousLotto() {
-        return previousLotto;
-    }
-
-    LottoNumber getBonusNumber() {
-        return bonusNumber;
+    LottoWinnerType winLotto(Lotto lotto) {
+        int countOfDuplicate = lotto.countDuplicateValue(previousLotto);
+        if (LottoWinnerType.matchCountWithBonus(countOfDuplicate)) {
+            return LottoWinnerType.valueOf(countOfDuplicate, lotto.contains(bonusNumber));
+        }
+        return LottoWinnerType.valueOf(countOfDuplicate, false);
```

Comment:
```suggestion
        return LottoWinnerType.valueOf(countOfDuplicate,  lotto.contains(bonusNumber));
```
조금 비효율적이라고 생각하실 수 있지만 훨씬 간결한 코드라고 생각해요.

Reply:
아... 왜 눈에 안 보였을까요. 훨씬 간결해진 거 같습니다..😓

Reply:
그렇다면 Identifier 의 로직을 수정하는것이 바람직 하지 않을까요?

Identifier 가 요구사항을 만족하지 못하고 있고 WinningCondition에서 그 요구사항을 구현함은
잘못된 표현인것 같아요.

Reply:
제가 `identifier` 클래스 로직을 살펴보니 말씀하신대로 수정해도 동작하도록 메소드들이 작성되어 있네요. 착각했습니다. 하지만 덕분에 피드백을 들어서 더 기억에 남을 거 같습니다.😅 감사합니다! 머지 부탁드려도 될까요?